aexpect/remote.py

```diff
@@ -17,6 +17,12 @@
 from aexpect import rss_client
 
 
+#: prompt to be used for shell sessions on linux machines (default)
+PROMPT_LINUX = r"^\\[.*\\][\\#\\$]\\s*$"
```

Comment:
Here you probably combined multiple escapes and regexp. If you want to write it as a regexp, you should use single `\`, if you want to write it as string, use multiple `\\`, but this does not work for me. Tested by:

```
>>> a = aexpect.remote.remote_login("ssh", "192.168.122.162", 22, "root", "redhat", aexpect.remote.PROMPT_LINUX, verbose=True)
Traceback (most recent call last):
  File "/home/medic/Work/Projekty/avocado/aexpect/aexpect/remote.py", line 266, in handle_prompts
    timeout=timeout, internal_timeout=0.5)
  File "/home/medic/Work/Projekty/avocado/aexpect/aexpect/client.py", line 947, in read_until_last_line_matches
    print_func)
  File "/home/medic/Work/Projekty/avocado/aexpect/aexpect/client.py", line 866, in read_until_output_matches
    raise ExpectTimeoutError(patterns, output)
aexpect.exceptions.ExpectTimeoutError: Timeout expired while looking for patterns ['[Aa]re you sure', '[Pp]assword:\\s*', '\\(or (press|type) Control-D to continue\\):\\s*$', '[Gg]ive.*[Ll]ogin:\\s*$', '(?<![Ll]ast )[Ll]ogin:\\s*$', '[Cc]onnection.*closed', '[Cc]onnection.*refused', '[Pp]lease wait', '[Ww]arning', '[Ee]nter.*username', '[Ee]nter.*password', '[Cc]onnection timed out', '^\\\\[.*\\\\][\\\\#\\\\$]\\\\s*$', 'Escape character is.*']    (output: '[root@localhost ~]# ')
>>> a = aexpect.remote.remote_login("ssh", "192.168.122.162", 22, "root", "redhat", "^\\[.*\\][\\#\\$]\\s*$", verbose=True)
>>> 
```

Reply:
Good catch! Fixed.